docs/content/manual/pre-release/node/node-deletion.md

```diff
@@ -27,3 +27,8 @@ title: Test node deletion
     4. Crash all replicas of the 3rd volume and trigger the auto salvage. --> The auto salvage should work. The volume works fine and the data is correct after the salvage.
```

Comment:
Could we give an example for "Crash all replicas" such as delete the IM-r pods?

Reply:
Sure, I will add more details to this step.